pkg/charm/instance.go

```diff
@@ -9,6 +9,7 @@ import (
 
 var (
 	HelpFlag   = "h"
+	HelpLongFlag   = "help"
```

Comment:
Thanks @zmajeed! Our Go developers will surely pass through to scrutinize the important parts of the code, but just to get one nit out of the way, could you align these up like below to make the linter in our CI happy? 😄 

```
        HelpFlag     = "h"
        HelpLongFlag = "help"
        HiddenFlag   = "hidden"
```